### PR TITLE
Percentage visitor wins

### PR DESCRIPTION
**Pull request by araspaas**
added percentage visitor wins method and find visitor wins method and passed tests
@sciencefixion @rrabinovitch @gabichuelas

### CODE REVIEW

**Review by gabichuelas — APPROVED** [review]
Looks good!

**Review by sciencefixion — APPROVED** [review]
Looks great! Quite concise execution!

File: lib/stat_tracker.rb [hunk 1]
```diff
@@ -40,6 +40,17 @@ def percentage_home_wins
     percentage.round(2)
   end
 
+  def find_visitor_wins
```
Comment (rrabinovitch) [inline lib/stat_tracker.rb:43]: nice job w this helper method

**Review by rrabinovitch — APPROVED** [review]
looks good!

**Pull request merged**